src/sandstorm/run-bundle.c++

```diff
@@ -992,6 +1003,30 @@ public:
     context.exitInfo(kj::str("reset OAuth configuration"));
   }
 
+  kj::MainBuilder::Validity adminToken() {
+    changeToInstallDir();
+
+    // Get 20 random bytes for token.
+    kj::byte bytes[20];
+    kj::FdInputStream random(raiiOpen("/dev/urandom", O_RDONLY));
```

Comment:
Use libsodium randombytes_buf(). (We should be switching over to that everywhere.)